### PR TITLE
1247 Textarea testing

### LINKED ISSUE #1247: TextArea Testing
### Dependency
- [x] #1267

### Overview
Testing is an important part of the development process that has been ignored in the Expunge Assist process thus far. To create a better protected codebase, we want to retroactively implement tests.

Create tests for the below scenarios. Feel free to include additional tests if you find opportunities for more coverage for unit tests of this textArea component.

### Action Items
- [x] text area sets up correctly
   - [x] text area renders
   - [x] text area displays the correct default value
- [x] props passed correctly
   - [x] placeholder is passed
   - [x] shows the correct number of rows
   - [x] passed correct id
   - [x] passed correct label
   - [x] passes disabled correctly
- [x] textArea is accessible
   - [x] is focusable

### Resources/Notes
[Jest](https://jestjs.io/docs/getting-started)
[React Testing Library](https://testing-library.com/docs/react-testing-library/intro/)

### PR DESCRIPTION
**Pull request by eburdekin**
Fixes #1247 

### Changes made: 
Added tests for the following:
- textarea renders and displays the correct default value
- textarea props are passed correctly
- textarea is accessible and focusable

### Reason for changes:
We want to retroactively implement testing to better protect our codebase

### CODE REVIEW

File: products/statement-generator/src/tests/textareas.test.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+    // label is associated with the textarea
+    const label = getByText('accessibleTextarea');
+    expect(label).toBeInTheDocument();
+    expect(label).toHaveAttribute('for', 'accessible-textarea');
+
+    // textarea is focusable
+    const input = getByPlaceholderText('Type here');
+    userEvent.click(input);
+    expect(input).toHaveFocus();
```
Comment (sydneywalcoff) [inline products/statement-generator/src/tests/textareas.test.tsx:109]: There are other ways to focus on a text area, like tabbing from the a different element. Our tests should cover that as well.

**Review by sydneywalcoff — CHANGES_REQUESTED** [review]
@eburdekin Hey Eileen!

These look great I had one small suggestion below!

**Comment by eburdekin** [comment]
Thanks @sydneywalcoff ! Added tests to account for tabbing onto the elements.

**Review by sydneywalcoff — APPROVED** [review]
@eburdekin Looks great! Thank you for making that accessibility addition!

**Pull request merged**